docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,146 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+**coala** has an awesome testing framework so you can write tests for bears
+with ease.
+
+You can use the following to test your bears:
+
+- ``LocalBearTestHelper.check_validity``
+- ``LocalBearTestHelper.check_results``
+- ``verify_local_bears``
+
+Understanding through examples
+------------------------------
+
+**EXAMPLE 1** using ``verify_local_bear``
+
+PyDocStyleBearTest
```

Comment:
could we have simpler `custom bears`  for the example?, this seems like a lot of work, for a newcomer, where they'd have to go through what is `PyDocStyleBear` in the first place